migrate/migrations/20200610183245-initial_tables.sql

```diff
@@ -0,0 +1,132 @@
+-- +migrate Up
+
+CREATE TABLE _auth_user
+(
+    id            text PRIMARY KEY,
+    app_id        text                        NOT NULL,
+    created_at    timestamp without time zone NOT NULL,
+    updated_at    timestamp without time zone NOT NULL,
+    last_login_at timestamp without time zone,
+    metadata      jsonb
+);
+
+CREATE SEQUENCE _auth_event_sequence;
+
+CREATE TABLE _auth_password_history
+(
+    id        text PRIMARY KEY,
+    app_id    text                        NOT NULL,
+    user_id   text                        NOT NULL REFERENCES _auth_user (id),
+    password  text                        NOT NULL,
+    logged_at timestamp without time zone NOT NULL
+);
+
+CREATE TABLE _auth_oauth_authorization
+(
+    id         text PRIMARY KEY,
+    app_id     text                        NOT NULL,
+    client_id  text                        NOT NULL,
+    user_id    text                        NOT NULL REFERENCES _auth_user (id),
+    created_at timestamp without time zone NOT NULL,
+    updated_at timestamp without time zone NOT NULL,
+    scopes     jsonb                       NOT NULL,
+    UNIQUE (app_id, user_id, client_id)
```

Comment:
Explicitly create **EACH** unique index with `CREATE UNIQUE INDEX` so that the name is deterministic. 

We should introduce the down migration as well. In the down migration we can drop the unique index first.

Reply:
I don't think we need to drop the unique indexes explicitly, since dropping the table would drop them too?

Reply:
Let's add `down` command first and we can verify whether unique indexes are dropped along with the tables

Reply:
I just tried and indexes are dropped along with the tables.